Reference/Searching/Examine/Quick-Start/index.md

```diff
@@ -66,208 +103,162 @@ Umbraco ships with three indexes:
 
 ([You can create your own indexes too](../indexing)) if you need to analyse text in a different language for example.
 
-You use a convenient service named the `IExamineManager` to retrieve first the Index by its 'alias' and then use the Index to get a reference to the Searcher. First, inject the `IExamineManager`:
-
-```
-@using Examine
-@inject IExamineManager ExamineManager;
-```
- and then get a reference: 
-
+You use a convenient service named the `IExamineManager` to retrieve first the Index by its 'alias' and then use the Index property so we can use the Searcher.
+Lets start by dependency injection this into our Service!
 ```csharp
- if(ExamineManager.TryGetIndex("ExternalIndex", out var index))
+private readonly IExamineManager _examineManager;
+
+public SearchService(IExamineManager examineManager)
 {
-    var searcher = index.Searcher;
+    _examineManager = examineManager;
+}
 ```
 ### Creating the Search Query
-With this in mind we begin to update the `else` condition with the following:
+With this in mind we begin to update the `SearchContentNames` method with the following:
 
 ```csharp
-else
+IEnumerable<string> ids = Array.Empty<string>();
+if (!string.IsNullOrEmpty(query) && _examineManager.TryGetIndex("ExternalIndex", out IIndex? index))
 {
-    //perform the search
-    //first we try to get the index, it is the ExternalIndex as we don't want to return unpublished things
-    //it returns the index in the var index
-    //be sure to add "@using Umbraco.Examine;" at the top of the view
-    if(ExamineManager.TryGetIndex("ExternalIndex", out var index))
-    {
-        var searcher = index.Searcher;
-        var results = searcher.CreateQuery("content").NodeTypeAlias("person").And().Field("nodeName", searchTerm).Execute();
-        if (results.Any())
-        {
-            <ul>
-                @foreach (var result in results)
-                {
-                    if (result.Id != null)
-                    {
-                        var node = Umbraco.Content(result.Id);
-                        <li>
-                            <a href="@node.Url()">@node.Name</a>
-                        </li>
-                    }
-                }
-            </ul>
-        }
-        else
-        {
-            <p>No results found for query @searchTerm</p>
-        }
-    }
-    return;
+    ids = index.Searcher.CreateQuery("content").NodeTypeAlias("person").And().Field("nodeName", query).Execute()
+        .Select(x => x.Id);
```

Comment:
It's probably easier to read and understand if we split the query into multiple lines here.

```suggestion
    ids = index
        .Searcher
        .CreateQuery("content")
        .NodeTypeAlias("person")
        .And()
        .Field("nodeName", query)
        .Execute()
        .Select(x => x.Id);
```